### PR TITLE
Add an inky focus behaviour for radio button/checkboxes

### PR DESCRIPTION
**Pull request by notwaldorf**
I'm not married to the name `PaperRadioButtonBehavior`, but I couldn't figure out a way to say it's got an inky state, and be in agreement with `PaperButtonBehavior`. Open to all suggestions.

This PR also cleans up the `bower.json`, which was not to style and also broken.

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata I made a thing! Please take a look! :)

File: demo/paper-radio-button.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../paper-ripple/paper-ripple.html">
+<link rel="import" href="../paper-radio-button-behavior.html">
+
+<dom-module id="paper-radio-button">
+
+  <style>
+  :host {
+    display: inline-block;
+    white-space: nowrap;
```
Comment (cdata) [inline demo/paper-radio-button.html]: Should we add some theme hooks? :dress:

File: demo/paper-radio-button.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  #offRadio {
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    border: solid 2px;
+    border-color: black;
```
Comment (cdata) [inline demo/paper-radio-button.html]: Should this be a variable?

File: demo/paper-radio-button.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  #onRadio {
+    position: absolute;
+    top: 4px;
+    left: 4px;
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    background-color: red;
```
Comment (cdata) [inline demo/paper-radio-button.html]: Should _this_ be a variable?

File: demo/paper-radio-button.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    -webkit-transform: scale(0);
+    transform: scale(0);
+    transition: -webkit-transform ease 0.28s;
+    transition: transform ease 0.28s;
```
Comment (cdata) [inline demo/paper-radio-button.html]: Maybe even *_this_* could be a variable??

**Comment by cdata** [comment]
Minor style questions, otherwise looks good :ship:

File: demo/paper-radio-button.html (hunk 4 above)
Comment (notwaldorf) [inline demo/paper-radio-button.html]: It's just the demo. I don't think anyone will reuse it :)

**Comment by notwaldorf** [comment]
Thanks! :sparkles:

**Pull request merged**